packages/react-dom/src/client/ReactDOMHostConfig.js

```diff
@@ -1164,32 +1213,179 @@ export function errorHydratingContainer(parentContainer: Container): void {
   }
 }
 
-export function getInstanceFromNode(node: HTMLElement): null | Object {
-  return getClosestInstanceFromNode(node) || null;
+export function isPersistentInstance(instance: Instance | Container): boolean {
+  if (instance.nodeType === ELEMENT_NODE) {
+    return isPersistentInstanceType(instance.tagName.toLowerCase());
+  }
+  return false;
```

Comment:
I would like to unify this with isPersistentInstanceType but sometimes I have a type string and other times I have a Container and we can't get the type from a Container in the reconciler so I think we need both :(